packages/ERTP/src/amountMath.chainmail

```diff
@@ -25,98 +26,92 @@ struct Extent {
    */
-  empty() -> (Units);
+  getEmpty() -> (Amount);
 
-  /** Return true if the Units is empty. Otherwise false. */
-  isEmpty(units :Units) -> (boolean);
+  /** Return true if the Amount is empty. Otherwise false. */
+  isEmpty(amount :Amount) -> (boolean);
 
-  /** Returns true if the leftUnits contains the rightUnits. */
-  includes(leftUnits :Units, rightUnits :Units) -> (boolean);
+  /** Returns true if the leftAmount is greater than or equal to the rightAmount. */
+  isGTE(leftAmount :Amount, rightAmount :Amount) -> (boolean);
 
   /** 
-   * Returns true if the leftUnits equals the rightUnits. We assume
-   * that if includes is true in both directions, equals is also true 
+   * Returns true if the leftAmount equals the rightAmount. We assume
+   * that if isGTE is true in both directions, isEqual is also true 
    */
-  equals(leftUnits :Units, rightUnits :Units) -> (boolean);
+  isEqual(leftAmount :Amount, rightAmount :Amount) -> (boolean);
 
   /**
-   * Returns a new units that includes both leftUnits and rightUnits.
+   * Returns a new amount that is the union of both leftAmount and rightAmount.
    *
-   * For fungible units this means adding the extents. For other kinds of
-   * units, it usually means including both.
+   * For fungible amount this means adding the extents. For other kinds of
+   * amount, it usually means including all of the elements from both
+   * left and right. 
    */
-  with(leftUnits :Units, rightUnits :Units) -> (Units);
+  add(leftAmount :Amount, rightAmount :Amount) -> (Amount);
 
   /**
-   * Returns a new units that includes the portion of leftUnits not included
-   * in rightUnits. If leftUnits doesn't include rightUnits, throw an error.
-   */
-  without(leftUnits :Units, rightUnits :Units) -> (Units);
+  * Returns a new amount that is the leftAmount minus the rightAmount
+  * (i.e. everything in the leftAmount that is not in the
+  * rightAmount). If leftAmount doesn't include rightAmount
+  * (subtraction results in a negative), throw  an error. Note: 
+  * because we throw an error if the result is "negative", this is NOT
+  * equivalent to set subtraction.
+  */
+  subtract(leftAmount :Amount, rightAmount :Amount) -> (Amount);
 }
 
 /**
- * The brand identifies the kind of assay, and has a function to get the
+ * The brand identifies the kind of issuer, and has a function to get the
  * alleged name for the kind of asset described. The alleged name (such
- * as 'BTC' or 'moola') is provided by the maker of the mint and should 
+ * as 'BTC' or 'moola') is provided by the maker of the issuer and should 
  * not be trusted as accurate.
  *
- * Every units created by the UnitOps will have the same brand, but recipients
- * cannot use the brand by itself to verify that a purported units is
+ * Every amount created by AmountMath will have the same brand, but recipients
+ * cannot use the brand by itself to verify that a purported amount is
  * authentic, since the brand can be copied. 
```

Comment:
Because its salient property is object identity, it cannot be copied. Or rather, a copy would fail to impersonate the original brand. How about "... since the brand can be reused by a misbehaving issuer".

Reply:
Yup, I mean copied in a non-programming sense. That's a good edit.